tests/guardrails/correctionRangeSchedule.tests.js

```diff
@@ -1,122 +1,97 @@
+const name = require('./testNames');
+
 module.exports = (test) => {
     var screen;
-    var settings;
+    var therapyScreen;
```

Comment:
[nit] `therapySettingsScreen`? (Last message about this. Assume this applies to all instances throughout code.)